src_pp/sec_double_int_volume.ptx

```diff
@@ -305,344 +286,345 @@
+    <me>
+      \ds \sum_{i=1}^n f(x_i,y_i)\dx\dy
+    </me>.
+  </p>
+
+  <p>
+    This approximates the signed volume under <m>f</m> over <m>R</m>.
+    As we have done before,
+    to get a better approximation we can use more rectangles to approximate the region <m>R</m>.
+  </p>
+
+  <p>
+    In general, each rectangle could have a different width <m>\dx_j</m> and height <m>\dy_k</m>,
+    giving the <m>i^\text{ th }</m> rectangle an area <m>\Delta A_i = \dx_j\dy_k</m> and the
+    <m>i^\text{ th }</m> rectangular solid a volume of <m>f(x_i,y_i)\Delta A_i</m>.
+    Let <m>\norm{\Delta A}</m> denote the length of the longest diagonal of all rectangles in the subdivision of <m>R</m>;
+    <m>\norm{\Delta A}\to 0</m> means each rectangle's width and height are both approaching 0.
+    If <m>f</m> is a continuous function,
+    as <m>\norm{\Delta A}</m> shrinks
+    (and hence <m>n\to\infty</m>)
+    the summation <m>\ds \sum_{i=1}^n f(x_i,y_i)\Delta A_i</m> approximates the signed volume better and better.
+    This leads to a definition.
+  </p>
+  <aside>
     <p>
-      The volume of the rectangular solid whose base is the
-      <m>i^\text{ th }</m> subregion and whose height is
-      <m>f(x_i,y_i)</m> is <m>V_i=f(x_i,y_i)\dx\dy</m>.
-      Such a solid is shown in <xref ref="fig_double_intro">Figure</xref>(b).
-      Note how this rectangular solid only approximates the true volume under the surface;
-      part of the solid is above the surface and part is below.
+      Recall that the integration symbol
+      <q><m>\int</m></q> is an <q>elongated S,</q>
+      representing the word <q>sum.</q>
+      We interpreted <m>\int_a^bf(x)\ dx</m> as <q>take the
+      <em>sum</em> of the areas of rectangles over the interval <m>[a,b]</m>.</q>
+      The double integral uses two integration symbols to represent a <q>double sum.</q>
+      When adding up the volumes of rectangular solids over a partition of a region <m>R</m>,
+      as done in <xref ref="fig_double_intro">Figure</xref>,
+      one could first add up the volumes across each row
+      (one type of sum),
+      then add these totals together
+      (another sum),
+      as in
+      <me>
+        \sum_{j=1}^n\sum_{i=1}^mf(x_i,y_j)\Delta x_i\Delta y_j
+      </me>.
     </p>
 
     <p>
-      For each subregion <m>R_i</m> used to approximate <m>R</m>,
-      create the rectangular solid with base area <m>\dx\dy</m> and height <m>f(x_i,y_i)</m>.
-      The sum of all rectangular solids is
+      One can rewrite this as
       <me>
-        \ds \sum_{i=1}^n f(x_i,y_i)\dx\dy
+        \sum_{j=1}^n\left(\sum_{i=1}^mf(x_i,y_j)\Delta x_i\right)\Delta y_j
       </me>.
     </p>
 
     <p>
-      This approximates the signed volume under <m>f</m> over <m>R</m>.
-      As we have done before,
-      to get a better approximation we can use more rectangles to approximate the region <m>R</m>.
+      The summation inside the parenthesis indicates the sum of heights <times/> widths,
```

Comment:
Same thing here: line numbers are off in the diff. But the file looks fine when viewed.